app/tests/src/test_format.py

```diff
@@ -264,35 +267,32 @@ def test__return_citation_link():
     chunk_list[1].document = doc[1]
     chunk_list[1].page_number = 3
 
-    bem_link = return_citation_link(chunk_list[0], "BEM")
+    bem_link = BemFormattingConfig().return_citation_link(chunk_list[0])
 
     assert "Open document to page 3" in bem_link
     assert "Source" not in bem_link
 
-    web_link = return_citation_link(chunk_list[1], "EDD")
+    web_link = FormattingConfig().return_citation_link(chunk_list[1])
     assert "page 3" not in web_link
     assert "Source" in web_link
 
 
-def test__format_web_subsections(chunks_with_scores):
+def test_build_accordions(chunks_with_scores):
     subsections = to_subsections(chunks_with_scores)
 
-    assert format_web_subsections(0, 0, chunks_with_scores, subsections, "") == "<div></div>"
+    config = FormattingConfig()
+    assert build_accordions(subsections, "", config) == "<div></div>"
     assert (
-        format_web_subsections(0, 0, [], [], "Non-existant citation: (citation-0)")
+        build_accordions([], "Non-existant citation: (citation-0)", config)
         == "<div><p>Non-existant citation: </p></div>"
     )
 
     assert (
-        format_web_subsections(0, 0, [], [], "List intro sentence: \n- item 1\n- item 2")
+        build_accordions([], "List intro sentence: \n- item 1\n- item 2", config)
         == "<div><p>List intro sentence: </p>\n<ul>\n<li>item 1</li>\n<li>item 2</li>\n</ul></div>"
     )
 
-    chunks_with_scores[0].chunk.document.name = "Your State Disability Insurance (SDI)"
-    chunks_with_scores[1].chunk.document.name = "Your State Disability Insurance (SDI): Another"
```

Comment:
This refactoring revealed that `chunks_with_scores` is not relevant to `format_web_subsections` (now `build_accordions`)